ControlMeasurements/Models/Measurement.cs

```diff
@@ -8,5 +8,10 @@ public class Measurement
         public PlaceType PlaceType { get; set; }
         public MeasurementType MeasurementType { get; set; }
         public int Value { get; set; }
+        public DateTime MeasurementDate { get; set; }
+        public Measurement()
```

Comment:
Po co ten konstruktor? Chcesz żeby data uzupełniała się automatycznie?

Reply:
Zmieniłem na automatyczne wypełnianie

Reply:
No i na razie spoko :) Potem to będzie miejsce, które można usprawnić. Co jeśli zrobiłeś pomiary wczoraj, ale wpisujesz je do systemu dzisiaj? Zawsze można zrobić rozwiązanie, które będzie miało zalety obu wersji -> np. pokazujesz kontrolkę z datą, którą można edytować, ale domyślnie znajduje się już w niej aktualna data. Ale to na przyszłość :)

Reply:
No tak masz racje